### PR TITLE
Slovak translation

### PR DESCRIPTION
**Pull request by ujovlado**
Slovak translation of Semver. I think it can be merged, but I will ask to proofread some folks.

### CODE REVIEW

**Comment by ujovlado** [comment]
@rootpd since you're one of my favourite grammar nazi :smiley: I'm kindly asking if you could check this translation

File: lang/sk/index.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+Zhrnutie
+--------
+
+Ak sa číslo verzie skladá podľa formátu MAJOR.MINOR.PATCH, tak zväčšujeme číslo:
+
+1. MAJOR verzie, keď sme spravili zmeny, ktoré nie sú spätne kompatibilné,
+2. MINOR verzie, keď sme pridali funkcionalitu so zachovaním spätnej
+kompatibility,
+3. PATCH verzie, keď sme opravili chyby a ostala zachovaná spätná komaptibilita.
```
Comment (rootpd) [inline lang/sk/index.md]: typo: _komaptibilita_

File: lang/sk/index.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+Značenie predbežných verzií a pridávanie metadát verzií je možné rozšírením
+spomínaného formátu MAJOR.MINOR.PATCH o potrebné informácie.
+
+Úvod
+----
+
+Vo svete softvéru, jeho vývoja a správy existuje hrozné miesto nazývané peklo
+závislostí *(angl. dependency hell)*. Znamená to, že čím viac váš systém rastie
+a máte v ňom integrovaných čoraz viac knižníc, tým je väčšia pravdepodobnosť,
```
Comment (rootpd) [inline lang/sk/index.md:27]: proposal: ".. a čím viac knižníc v ňom integrujete ..."?

File: lang/sk/index.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+že sa ocitnete v tomto pekle a budete kompletne zúfalí.
```
Comment (rootpd) [inline lang/sk/index.md]: proposal: "že sa kompletne zúfalí v tomto pekle ocitnete" (znie mi to zdravšie, a končím s ťažkým nacizmom :))

File: lang/sk/index.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+
+Ak majú systémy veľa závislostí, razdva sa vám stane, že vydávanie novej sa
```
Comment (rootpd) [inline lang/sk/index.md]: typo: _razdva_ (myslim ze s pomlckou alebo oddelene)

File: lang/sk/index.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+stane nočnou morou. Ak je špecifikácia závislostí príliš striktná, hrozí vám, že
+ostanete na jednej verzii *(angl. version lock)* a nebudete mať možnosť prejsť
+na ďalšiu verziu bez toho aby ste vydali ďalšie verzie všetkých balíčkov, na
```
Comment (rootpd) [inline lang/sk/index.md]: bez toho, aby (chyba ciarka).. spravim rovno PR? :D

Comment (ujovlado) [inline lang/sk/index.md]: @rootpd ak sa ti chce, sprav PR sem https://github.com/rekurzia/semver.org, potom to re-pushnem. Alebo aj hociako inak mozes - len aby sa to dalo mergnut, pripadne cherry-picknut. :wink:

**Comment by ujovlado** [comment]
@tiso do you also have some time to proofread? :wink:
(if you'll find many mistakes, please create PR to this source https://github.com/rekurzia/semver.org, I'll handle it)

**Comment by ujovlado** [comment]
@tiso looks great, thanks! :+1:

**Comment by ujovlado** [comment]
@Haacked Slovak translation was reviewed and is prepared. Please, consider merging this. :wink: 
(if here are too many commits, I can re-push them to single one)
Thanks!

**Comment by haacked** [comment]
selfie-0

**Pull request merged**